src/pyoauth/oauth/google.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import urllib
+from urllib.parse import urlencode
+from urllib.request import Request, urlopen
+
+from base import OAuth
+from src.pyoauth.utils.oauth import get_query_auth, get_query_access
+
+_client_id = os.environ.get("GOOGLE_CLIENT_ID")
+_client_secret = os.environ.get("GOOGLE_CLIENT_SECRET")
+
+# Endpoints for Google OAuth
+_auth_endpoint = "https://accounts.google.com/o/oauth2/v2/auth"
```

Comment:
Endpoints can be accessed from the OIDC configuration endpoint and this will eliminate the need to refactor the code if endpoints got changed in the future.

Reply:
Will be done in next PR